EnhancedMediaPlayer/MediaPlayerTappableView.swift

```diff
@@ -0,0 +1,132 @@
+// MediaPlayerTappableView.swift
+
+import SwiftUI
+
+struct MediaPlayerTappableView: View {
+    @ObservedObject var viewModel: ViewModel
+
+    var body: some View {
+        ZStack {
+            renderSeekLayer()
+            renderControlsLayer()
+        }
+    }
+
+    @ViewBuilder func renderControlsLayer() -> some View {
```

Comment:
Rename these to `ControlsTrigger` instead of `Controls`. You're not actually rendering controls, you're rendering the trigger to toggle the controls. Same for the action name, etc